### PR TITLE
Feature/distributions

### PR DESCRIPTION
**Pull request by dcorquir**
Create endpoint to distributions

### CODE REVIEW

File: src/api.py [hunk 1]
```diff
@@ -3,10 +3,12 @@
 
 from src.config import fastApiConfig
 from src.db import MongoEngine
-from src.routes import units_routes
-
+from src.routes import (
+    units_routes, 
+    distributions_routes)
```
Comment (davisfelipe) [inline src/api.py:8]: set line break before parentheses

File: src/utils/units.py [hunk 2]
```diff
@@ -13,4 +19,13 @@ def distance(cls):
...
+        return cls.units_length
+    
+    @classmethod
+    def time(cls):
+        '''
+            Get units of time
+        Return:
+            List: Time units list
+        '''
+        return cls.units_time
```
Comment (davisfelipe) [inline src/utils/units.py:31]: Line brake at the end of the file

**Review by davisfelipe — CHANGES_REQUESTED** [review]
I made comments on your code.

File: src/routes/distributions.py [hunk 3]
```diff
@@ -1,58 +1,106 @@
...
\ No newline at end of file
+
+
+@distributions_routes.get("/distributions/disease_state")
+def list_disease_state_distribution():
+    """
+    Get disease states distribution list
+
+    Return:
+        Disease state distributions object list
```
Comment (Camilo-HG) [inline src/routes/distributions.py:85]: Change: "Disease state" ---> "Disease states"

**Review by Camilo-HG — CHANGES_REQUESTED** [review]

**Pull request merged**